src/main/java/emissary/util/ImageIOClass.java

```diff
@@ -0,0 +1,75 @@
+package emissary.util;
+
+import java.lang.reflect.Constructor;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.imageio.spi.IIORegistry;
+
+import emissary.config.ConfigEntry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class to support loading the imported ImageIO classes. This behavior was moved here due to the timing issues seen
+ * when attempting to initialize these classes in the "Places" classes.
```

Comment:
IT would be nice to describe the distinction between an `imageIOClass` and a `surplusImageIOClass` somewhere in the Javadoc. It is not immediately clear from looking at the code why there are two separate groups of classes

Reply:
I added a comment in the class javadoc explaining what imageIOClass and surplusImageIOClass are.